### DISCUSSION TITLE
Can we setup Pimcore Direct Edit plugin locally on system?

### DISCUSSION THREAD

**Discussion by nirbanc2020** [pimcore/pimcore#9425] [Q&A / Support]
After installing the Direct Edit plugin in Pimcore X,I setup mercure. But after setting all configurations in pimcore,When I enter the url 'https://your-app-domain.com/hub/.well-known/mercure' in browser,nothing opening.Can anyone please guide how to setup the same?

**Comment by fashxp** [comment]
Seems mercure is not setup or running properly. Normally a response telling you something like `missing topic` should be shown in browser then. 

Please contact your sales contact person at Pimcore to get access to the corresponding github repositories.
There we are happy to help you further. 
Thanks.

**Reply by nirbanc2020** [reply]
But When I checked the same,It is active and running.
![sec](https://user-images.githubusercontent.com/73533273/121852406-11b0cb00-cd0d-11eb-8aa3-663f4de2ab83.png)

**Reply by fashxp** [reply]
then maybe your reverse-proxy is not forwarding the requests to mercure properly...

**Reply by nirbanc2020** [reply]
Ok..I have created the reverse proxy in virtual host file.Is it correct or can you guide me if something is missing? I am trying to setup in local system.
![jjjj](https://user-images.githubusercontent.com/73533273/121853434-8c2e1a80-cd0e-11eb-8d94-6543b6ad2199.png)

**Reply by nirbanc2020** [reply]
After installing Pimcore Direct Edit bundle ,When I installed the Pimcore Direct Edit Client on Windows machine ,while opening any asset through the button "Open To Edit Locally" I am getting the message "Cannot connect to event - server. Please try again in a couple of minutes and do not forget to reload your browser." Can anyone help regarding the same?

**Reply by fashxp** [reply]
the problem is, that mercure is not accessible from the client ...

**Reply by nirbanc2020** [reply]
I run https://your-app-domain.com/hub/.well-known/mercure the link in browser I am getting Missing "Topic" as output.Any thing more I have to do for accessing mercure through client?

**Reply by fashxp** [reply]
please have a look at the console output of your browser ... are there any error messages?

**Reply by nirbanc2020** [reply]
I checked the same.I am getting "Content Security Policy: The page’s settings blocked the loading of a resource at inline (“default-src”)." in console. no more error message I am getting

**Reply by fashxp** [reply]
that might be the problem ... on our demo, this error is not showing up ... and is it is connected to event server 

![image](https://user-images.githubusercontent.com/8792145/122372840-4b7a0f80-cf61-11eb-9160-af02da558c82.png)

maybe you are not on https thus chrome is blocking?
